onboarding/src/screenshotTest/kotlin/io/homeassistant/companion/android/onboarding/welcome/WelcomeScreenshotTest.kt

```diff
@@ -0,0 +1,18 @@
+package io.homeassistant.companion.android.onboarding.welcome
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.PreviewLightDark
+import com.android.tools.screenshot.PreviewTest
+import io.homeassistant.companion.android.compose.HAThemeScreenshot
+
+class WelcomeScreenshotTest {
+
+    @PreviewTest
+    @PreviewLightDark
```

Comment:
I think `@HAPreviews` is a good candidate here to make sure we don't forget about larger screens (where you might get very wide text or empty layouts otherwise).